units/host_allocator.go

```diff
@@ -115,11 +115,12 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 		return
 	}
 
-	existingHosts, err := host.AllRunningHosts(j.DistroID)
+	existingHosts, err := host.AllActiveHosts(j.DistroID)
 	if err != nil {
 		j.AddError(errors.Wrap(err, "Database error retrieving running hosts"))
 		return
 	}
+	upHosts := existingHosts.Uphosts()
```

Comment:
Don't you need to add this to the log statement too?

Reply:
nope! the uphosts are a superset of the existing hosts, and the Stats() method does the right kind of accounting in the log message.